x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/related_detection_rules_callout.test.tsx

```diff
@@ -32,15 +29,15 @@ describe('Policy form RelatedDetectionRulesCallout component', () => {
 
     expect(renderResult.getByTestId('test')).toHaveTextContent(
       exactMatchText(
-        'View related detection rules. Prebuilt rules are tagged “Elastic” on the Detection Rules page.'
+        'The Endpoint Security detection rule is enabled automatically with Elastic Defend. This rule must remain enabled to receive endpoint alerts. Learn MoreExternal link(opens in a new tab or window).'
```

Comment:
```suggestion
        'The Endpoint Security detection rule is enabled automatically with Elastic Defend. This rule must remain enabled to receive Endpoint alerts. Learn MoreExternal link(opens in a new tab or window).'
```

Reply:
Made the change here: https://github.com/elastic/kibana/pull/172471/commits/3102e34a843918d0789908e310aa672ec132e7f3